src/templates/ru/2019/error.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base/2019/error.html" %}
+
+{% block date_published %}2019-11-04T12:00:00.000Z{% endblock %}
+{% block date_modified %}2020-08-06T00:00:00.000Z{% endblock %}
+
+{% block unknown_error %}Неизвестная ошибка{% endblock %}
+
+{%
+  set localizedStatusCode = {
+    400: "Неверный запрос",
+    404: "Страница не найдена",
+    500: "Внутренняя ошибка сервера",
+    502: "Временная ошибка сервера"
+  }
+%}
+{% macro expandedTitle(errorTitle) %}Ошибка: {{ errorTitle }}{% endmacro %}
+{%
+  set localizedErrorMessage = {
+    "Unsupported year requested": "Запрошен несуществующий год",
+    "Unsupported chapter requested": "Запрошен несуществующая глава",
+    "Unsupported language requested": Запрошен несуществующий язык",
```

Comment:
Missing opening quote:

```suggestion
    "Unsupported language requested": "Запрошен несуществующий язык",
```